packages/amplify-graphql-model-transformer/src/resolvers/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './generators';
 export * from './common';
+export * from './rds';
```

Comment:
Why are we now exporting this? (I'm assuming it's to make new logic in `resolver.ts` easier to access somewhere?

Reply:
I exported `createLayerVersionCustomResource`, `createSNSTopicARNCustomResource` for the unit test in the same package:
`import { createLayerVersionCustomResource, createSNSTopicARNCustomResource } from '@aws-amplify/graphql-model-transformer';
`
I guess define a relative path instead of exporting is a better choice here?

Reply:
No need to change, as long as we don't need to export it to the public interface. I was just wondering why the new export :)